api/app/controllers/project.py

```diff
@@ -0,0 +1,27 @@
+from typing import List
+from uuid import UUID
+from sqlalchemy.orm import Session
+from app.models.project import Project
+from app.schemas.project_schemas import ProjectCreate, ProjectRead
+
+
+class ProjectController:
```

Comment:
I'm wondering... for "creatable" objects - projects, chats, messages - do we want "created_by" and "last_updated_by" fields, like we have in SWIS? Could create an extended Base `AuditedBase` or something like that with these columns added as optional uuid values. that'll be populated by the requesting user, and then the audit table will give us a "who changed what when" log we can display which is pretty sick

Reply:
agreed

Reply:
I'll add in the next pr